test/core/test/link.js

```diff
@@ -400,5 +400,96 @@
             assert.isTrue(range.endContainer.equals(range.endContainer));
             assert.strictEqual(range.endOffset, 1);
         });
+
+        describe('when appendProtocol is false', function() {
+
+            it('should not add default protocol when creating a link', function() {
+                if (CKEDITOR.env.ie) {
```

Comment:
Hey @antoniopol06, just out of curiosity... is this test really failing in IE? Is the feature working? Same for the others below...